tools/topology/topology2/platform/intel/deep-buffer.conf

```diff
@@ -46,3 +50,55 @@ Object.Base.route [
 		sink	gain.$DEEP_BUFFER_PIPELINE_ID.1
 	}
 ]
+
+# Spawn another instance
+IncludeByKey.DEEP_BUF_2 {
+"true"	{
+		Object.Pipeline.deepbuffer-playback [
+			{
+				index $DEEP_BUFFER_PIPELINE_ID_2
```

Comment:
where is this defined? Don't you need defaults for this and all the *_2's below?

Reply:
I am thinking to add those in the parent conf (ie cavs_rt5682.c) in another patch:

+	DEEP_BUFFER_PIPELINE_ID_2	35
+	DEEP_BUFFER_PCM_ID_2		51
+	DEEP_BUFFER_PIPELINE_SRC_2		'mixin.35.1'
+	DEEP_BUFFER_PIPELINE_SINK_2	'mixout.4.1'
+	DEEP_BUFFER_PCM_NAME_2		'Deepbuffer Amps'